tests/antithesis/Makefile

```
@@ -0,0 +1,5 @@
+REPOSITORY_ROOT := $(shell git rev-parse --show-toplevel)
+
+.PHONY: client
+client:
```

Because we may want to include this in the top-level `Makefile`, I suggest it have the `anthesis` prefix. And perhaps a more descriptive name.

```suggestion
.PHONY: antithesis-build-client-docker-image
antithesis-build-client-docker-image:
```

---

done!